zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -85,6 +86,7 @@
 public class ZkClient implements Watcher {
   private static final Logger LOG = LoggerFactory.getLogger(ZkClient.class);
 
+  public static final long DEFAULT_TTL = -1L;
```

Comment:
Nit: Would `TTL_NOT_SET` be a more descriptive name for this? Also, I see this is a `public` constant - are you planning to use this in other classes?

Reply:
Yes, that's a better name. I'll change it.
Also yes, it will be used in other classes such as BaseDataAccessors and ZkClient implementations. I was thinking of creating a separate class for this constant, but since it was only one variable I thought I'd just put it in ZkClient. Let me know if a separate constants class is preferred and I can add it.